### DISCUSSION TITLE
Issues pairing EcoDim 0.5 Zigbee duo dimmer

### DISCUSSION THREAD

**Discussion by TMV-NL** [Koenkk/zigbee2mqtt#10920] [Devices]
Hi,
I cannot get my EcoDim to work in Z2M.
1. I put the converter in the Z2M folder https://github.com/Koenkk/zigbee-herdsman-converters/blob/89fd37d7fc0715e62fb6e62e83a9fc48cbb31fba/devices/ecodim.js
2. I amended the config file to add the converter. 
(I tried both with leaving and removing all the quotes in the config file)

Result is always the same: config file is set to the default, device remains unsupported.

Any suggestions?

**Comment by TMV-NL** [comment]
I noticed the issue is in the converter, because I can amend the config file and have it loaded, as long as the converter file is not added.

Also, if I run the converter through a JSON checker, I get a syntax error on position 1 already.

Could anybody give me some suggestions please?

**Reply by Koenkk** [reply]
Updating to the latest-dev branch should be more fault proof. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

**Comment by TMV-NL** [comment]
Hi Koen, thanks for the instruction.
I've no experience in Linux - so how do I run your instruction?
I tried from a SSH terminal via line instructions, but that is not correct.
________________________________
Van: Koen Kanters ***@***.***>
Verzonden: zaterdag 22 januari 2022 08:46
Aan: Koenkk/zigbee2mqtt ***@***.***>
CC: TMV-NL ***@***.***>; Author ***@***.***>
Onderwerp: Re: [Koenkk/zigbee2mqtt] Issues pairing EcoDim 0.5 Zigbee duo dimmer (Discussion #10920)


Updating to the latest-dev branch should be more fault proof. (https://www.zigbee2mqtt.io/advanced/more/switch-to-dev-branch.html)

—
Reply to this email directly, view it on GitHub<https://github.com/Koenkk/zigbee2mqtt/discussions/10920#discussioncomment-2021449>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXJPPVXCOUVYENK63GBY3ZLUXJOGBANCNFSM5MNWWEGQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Koenkk** [reply]
The instructions are the same when updating z2m, but before `git pull` do a `git checkout dev`

**Comment by TMV-NL** [comment]
Hi Koen,

When will this dev branche released?
I have no dev background nor Linux knowledge. I can execute instructions but I do not know what you mean as 'same  when updating z2m'. I'd expect a button in the UI.

Sorry for not being the regular user but your help is appreciated!
________________________________
Van: Koen Kanters ***@***.***>
Verzonden: zaterdag 22 januari 2022 14:43
Aan: Koenkk/zigbee2mqtt ***@***.***>
CC: TMV-NL ***@***.***>; Author ***@***.***>
Onderwerp: Re: [Koenkk/zigbee2mqtt] Issues pairing EcoDim 0.5 Zigbee duo dimmer (Discussion #10920)


The instructions are the same when updating z2m, but before git pull do a git checkout dev

—
Reply to this email directly, view it on GitHub<https://github.com/Koenkk/zigbee2mqtt/discussions/10920#discussioncomment-2022744>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/AXJPPVUXK2H4WOLCCDPLSUTUXKX75ANCNFSM5MNWWEGQ>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by Koenkk** [reply]
This will be included in the next release on 1 February.

**Reply by TMV-NL** [reply]
I noticed the release is delayed. When is the new release :-)

**Reply by Koenkk** [reply]
It has been released 2 days ago (https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.23.0)

**Reply by TMV-NL** [reply]
Ok... should it have triggered an update notice? It didn't.
Anyway, thanks

**Reply by TMV-NL** [reply]
And still 1.18 is is the version that is installed via HASS

**Reply by Koenkk** [reply]
If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Reply by TMV-NL** [reply]
Solved. Thanks for your help
